Client/Clique/Common/Constants/Constants.swift

```diff
@@ -6,13 +6,43 @@
 //
 
 import Foundation
+import Alamofire
+import UIKit
 
 enum Constants {
     
     enum Service {
         private static let baseURL = "https://cliquewebservice20221115180920.azurewebsites.net/api/"
         static let registerURL = baseURL.appending("Authentication/RegisterUser")
-        static let loginURL = baseURL.appending("Authentication/LoginUser")    
+        static let loginURL = baseURL.appending("Authentication/LoginUser")
+        static let profileGetUserURL = baseURL.appending("User")
+        static let profileUpdateURL = baseURL.appending("User/UpdateUserData")
+        static let passwordUpdateURL = baseURL.appending("User/UpdateUserPassword")
+        
+        static func getToken() -> String {
+            guard let token = UserStorage.token else { return "" }
+            return token
+        }
+        
+        static func authorizationHeader() -> HTTPHeaders {
+            let header: HTTPHeaders = [
+                "Authorization": "Bearer"
+                + " " + "\(getToken())",
+                "Accept": "application/json"
+            ]
+            return header
+        }
+    }
```

Comment:
What you can to here is to define to add authorization token only if token exists.
If token does not exist, no need to add `Authorization` key/value pair to header at all.

```swift
static func requestHeader() {
    var headers: HTTPHeaders = [
        "Accept": "application/json"
    ]

    If let token = UserStorage.token {
        headers["Authorization"] = "Bearer \(token)"
    }

    return headers
}
```